src/Swarm/Game/CESK.hs

```diff
@@ -350,39 +352,22 @@ prettyFrame (FDef x) = "def " ++ from x ++ " = _"
 prettyFrame FExec = "exec _"
 prettyFrame (FBind Nothing t _) = "_ ; " ++ prettyString t
 prettyFrame (FBind (Just x) t _) = from x ++ " <- _ ; " ++ prettyString t
-prettyFrame FImmediate {} = "(_ : cmd a)"
+prettyFrame (FImmediate c w r) = unwords ["immediate@" <> T.unpack (Syntax.syntax $ Syntax.constInfo c), show w, show r]
 prettyFrame (FUpdate loc) = "store@" ++ show loc ++ "(_)"
 prettyFrame FFinishAtomic = "finishAtomic"
 
 --------------------------------------------------------------
 -- Wrappers for functions in FImmediate
```

Comment:
We should get rid of this comment too, `FImmediate` no longer contains functions.

Reply:
Indeed, I am considering moving them to Robot and World respectively anyway. 🙂